### ISSUE TITLE
Sub Account Update (https PUT) not implemented 

### ISSUE THREAD

**Issue by andrew-oko-odion** [Flutterwave/rave-ruby#1]
Sub Account Update not implemented.  
I expect to be able to update a subaccount by its ID field

**Comment by temiadesina** [comment]
Hi @andrew-oko-odion this has been resolved with #2

**Comment by andrew-oko-odion** [comment]
Thanks you! 

But I do not think its finished, the method `update_subaccount` is not well documented in the `README`. It reads  

```
.update_subaccount(id)
This function is used to update a subaccount by taking in the id as its argument.

Sample update_subaccount call:
response = subaccount.update_subaccount("1234")

``` 

What about the payload.  The method documentation did not talk about how to pass in the payload. 
sorry this would not work! 

@timi did you even look at it?
